docassemble/AssemblyLine/data/questions/al_visual.yml

```diff
@@ -20,6 +20,10 @@ default screen parts:
     % if get_config('debug'):
     `id: ${ user_info().question_id }`  
     `Package: ${ user_info().package } ${ package_version_number }; ${ al_version }`
+    
+    [:save: Snapshot (dev)](${ url_ask(al_sessions_url_ask_snapshot) })
```

Comment:
Do we want to add this to everyone on debug servers, or should it be hidden under a different config, so it doesn't overload debug?

Reply:
What are your thoughts there? I don't want to add too many config options if possible, but I'm listening. debug should definitely be turned off for production servers--both for performance reasons and security.

Reply:
For our use cases, it seems fine. Not sure if other people using AL 1) use debug in the dev servers (or how many have the dev/test/prod distinction), and 2) would know what this feature is: I'm reminded of the bottom feedback / About bar causing issues with ILAO recently. 

Configs are kinda confusing though. There's no a lot of discoverability. IMO either a simple "session snapshot" config (can be undocumented for now) or a minor version bump to indicate a new feature would be a fine fix.